app/src/main/javascript/bundles/experiment-page/src/ExperimentPageRouter.js

```diff
@@ -24,18 +24,50 @@ const TabCommonPropTypes = {
 }
 
 // What component each tab type should render, coupled to ExperimentController.java
-const tabTypeComponent = {
-  'results' : TSnePlotViewRoute,
-  'experiment-design' : ExperimentDesignRoute,
-  'supplementary-information' : SupplementaryInformationRoute,
-  'downloads' : DownloadsRoute
+let tabTypeComponent = []
+
+function validateCommonRequiredProps({speciesName},{atlasUrl},{experimentAccession}) {
```

Comment:
I am a bit confused why we need this function at all.
We defined the types of the props in the definition of the prop types and we also defined whether they are required or not.
Why do we need another way to validate all of these again?